doc/conf_ref.rst

```diff
@@ -525,7 +525,21 @@ configuration file by your distribution to override the DNF defaults.
     ``"/etc/dnf/vars", "/etc/yum/vars"``. See :ref:`variable files <varfiles-label>`
     in Configuration reference.
 
-.. _conf_repo_options-label:
+.. _exclude_from_weak-label:
+
+``exclude_from_weak``
+    :ref:`list <list-label>`
+
+    Prevent installing packages specified by a name or a glob and separated by a comma to satisfy recommends
+    or supplements dependency. Defaults to ``[]``.
```

Comment:
I'd separate the description of how to specify the packages from the rest to improve the readability.

My suggestion: Prevent installing packages as weak dependencies (recommends or supplements). The packages can be specified by a name or a glob and separated by a comma. Defaults to ``[]``.

Reply:
Thanks, I used your suggested version.